pop.css

```diff
@@ -0,0 +1,165 @@
+.fondo-popup {
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(10px);
+}
```

Comment:
- [ ] Great job in implementing the changes suggested by the previous reviewer. However, I noticed that after fixing the issues you had with `image` pointed out by the previous reviewer it introduced a few issues which I will highlight in the below comment:
   - Your `see live` and `see source` buttons are not visible on large screen sizes
   -  Your project description `font-size` becomes very small on a large screen size and does match that of the `Figma` design.   
   
- Kindly consider fixing the issues mentioned above to help make your design look responsive, match that of the `Figma` design, and create a good user experience.  Below is a short screen recording of what the issue looks like:



https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/107192076/8c868266-a522-40f4-8f92-61e897236d94

Reply:
Thank you very much for helping me, for taking the time and providing clear examples of how to solve it.
All the changes that you have suggested to me as have been corrected satisfactorily 👍